src/main/java/com/powsybl/ws/commons/computation/service/AbstractWorkerService.java

```diff
@@ -141,7 +150,12 @@ public Consumer<Message<String>> consumeRun() {
             } catch (InterruptedException e) {
                 Thread.currentThread().interrupt();
             } catch (Exception e) {
-                if (!(e instanceof CancellationException)) {
+                if (isFailureWithLogs(e)) {
```

Comment:
I wonder why instead of modifying this lib, we don't handle this in handleNonCancellationException() in the appropriate service?